apps/build-onchain-apps/contracts/foundry.toml

```diff
@@ -0,0 +1,13 @@
+[profile.default]
+src = "src"
+out = "out"
+libs = ["lib"]
+fs_permissions = [{ access = "read", path = "./"}]
+
+[rpc_endpoints]
+base_goerli="${RPC_URl}"
```

Comment:
I'm not sure if something like this works in toml, but it seems like for this be a full solution we'd need the network name also to be dynamic. Otherwise we end up in the same place where if it's specific to a network we might as well hardcode the value for that. 

```suggestion
${NETWORK}="${RPC_URl}"
```

Reply:
Seems like it wont work
![image](https://github.com/coinbase/build-onchain-apps/assets/35871990/8b98e019-b418-47ef-b98b-a19053501ef4)

Reply:
Makes sense! I tried it and it works :) Updated it now